irohad/main/application.hpp

```diff
@@ -25,6 +25,8 @@ namespace iroha {
     class WsvRestorer;
     class TxPresenceCache;
     class Storage;
+    class ReconnectionStrategyFactory;
+    struct PoolWrapper;
```

Comment:
Probably can be removed.

Reply:
`ReconnectionStrategyFactory ` still has to stay.